### PR TITLE
DT-1958: ✨ DPS user edit email

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
           command: npx cypress install
       - run:
           name: Run the node app.
-          command: npm run start-feature --env=cypress.env
```
Comment (petergphillips) [inline .circleci/config.yml:61]: not needed since no groovy tests these days

File: backend/api/oauthApi.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const oauthApiFactory = (client, { clientId, clientSecret, url }) => {
   const currentRoles = (context) => get(context, '/api/user/me/roles')
   const getUserEmail = async (context, { username }) => {
     try {
-      return await get(context, `/api/user/${username}/email`)
+      return await get(context, `/api/user/${username}/email?unverified=true`)
```
Comment (petergphillips) [inline backend/api/oauthApi.js:39]: need unverified emails too otherwise an amend of the email won't then show it

File: package.json [hunk 3]
```diff
@@ -89,8 +89,8 @@
     "watch-ts": "tsc -w",
     "watch-node": "nodemon --watch dist/ dist/server.js | bunyan -o short",
     "start:dev": "npm run build && concurrently npm:watch-*",
-    "start-feature": "npm run copy-assets && npm run compile-sass && export $(cat $npm_config_env) && node dist/server.js",
-    "test-watch-node-feature": "export $(cat $npm_config_env) && nodemon --watch dist/ dist/server.js | bunyan -o short",
+    "start-feature": "npm run copy-assets && npm run compile-sass && export $(cat cypress.env) && node dist/server.js",
```
Comment (petergphillips) [inline package.json:92]: can hardcode `cypress.env` as no groovy tests these days

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**